### ISSUE TITLE
Edit the Current Ticket

### ISSUE THREAD

**Issue by engahmadkadi** [vocascan/vocascan-frontend#107]
ِAs a user of vocascan-frontend, I wish to have the feature of editing the shown ticket directly while learning. This eases the continuous enhancement of own vocabularies.

To edit the ticket, there must be an edit button on the (upper left) corner of both sides of the ticket. Once I click on it, editing fields should appear (on new window?). The edit fields have to be populated with the current values of the ticket. The user should have the options to either save his changes or to cancel the editing. The database should be updated with the new values, in case of submitting save.

**Comment by engahmadkadi** [comment]
Related discussion #103

**Comment by luwol03** [comment]
My idea of the implementation would be to easily reuse the vocab form (like it is already on editing in library view) and add a little button to the top of the learning window, the card or a different position. But lets start a little poll for that (just comment your preferences).

![image](https://user-images.githubusercontent.com/60048565/163224566-45b59bcb-9998-4e41-89a5-3d6c40ed974d.png)
![Unbenannt](https://user-images.githubusercontent.com/60048565/163225162-b49dfbaa-b8bf-48c6-99f3-e0ddebf66b77.png)

**Comment by engahmadkadi** [comment]
I had the same idea in mind, to reuse the functionality introduced at different place.

But to make it more clear to me: the pen icon will cause a pop-up window to appear, and after closing it, we stay at the same page i.e. the learning page were the user was. Right?

**Comment by luwol03** [comment]
Yes, after clicking on save the modal closes and the vocabulary is saved in background and the current card keeps there updated until you click to see the next card.
